gateway/app/drivers/qodProvisioning/nef.py

```diff
@@ -0,0 +1,290 @@
+import datetime
+import logging
+import uuid
+
+import httpx
+from pydantic import AnyUrl
+
+from app.interfaces.qodProvisioning import (
+    QoDProvisioningInterface,
+    ResourceNotFound,
+)
+from app.redis import get_redis
+from app.schemas.afSessionWithQos import (
+    AsSessionWithQoSSubscription,
+    UserPlaneNotificationData,
+)
+from app.schemas.commonData import (
+    FlowInfo,
+    Link,
+    UserPlaneEvent,
+)
+from app.schemas.qodProvisioning import (
+    CloudEvent,
+    CreateProvisioning,
+    Datacontenttype,
+    ProvisioningInfo,
+    RetrieveProvisioningByDevice,
+    Specversion,
+    Status,
+    StatusInfo,
+    Type,
+)
+from app.settings import settings
+import hashlib
+
+_prefix_info = "qodprovisioninginfo"
+_prefix_gateway_nef = "qodprovisioningnef"
+_prefix_device = "qodDevice"
+LOG = logging.getLogger(__name__)
+
+
+class NEFQoDProvisioningInterface(QoDProvisioningInterface):
+    def __init__(self) -> None:
+        super().__init__()
+        self.httpx_client = httpx.AsyncClient()
+        self.redis = get_redis()
+
+    async def create_provisioning(self, req: CreateProvisioning) -> ProvisioningInfo:
+        provisioning_id = uuid.uuid4()
+
+        payload = AsSessionWithQoSSubscription(
+            supportedFeatures="",
```

Comment:
```suggestion
            supportedFeatures="800820",
```

To enable support for the `gpsi` field